packages/shared/src/components/notifications/Toast.spec.tsx

```diff
@@ -49,8 +53,9 @@ it('should display a toast notification', async () => {
   fireEvent.click(button);
   await screen.findByRole('alert');
   await screen.findByText('Sample Notification');
-  await act(() => new Promise((resolve) => setTimeout(resolve, 1500))); // wait for the toast to expire
-  expect(screen.queryByRole('alert')).not.toBeInTheDocument();
+  await waitForElementToBeRemoved(() => screen.queryByRole('alert'), {
```

Comment:
This is amazing, I never knew this was supported.

Reply:
Yep, preferred way actually as it will deal with it's own cleanup :)